db/migrate/20170206211526_add_color_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddColorToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :color, :string
```

Comment:
We already have "white_player" and "black_player" attributes in the games table, so we don't need to add color to the users table.